src/components/TextSize.js

```diff
@@ -20,13 +20,13 @@ export const TextSize = ({ close }) => {
   return <div class='textsize'>
     <div class='header'>{i18n('header-textsize')}</div>
     <div class='content'>
-      <div class='preview adjustable-font-size'>
+      <div class={`preview font-size-${textSize}`}>
```

Comment:
Would it make sense to build the class name inside the hook since this has to be done everywhere?

Reply:
> Just sending random thoughts your way. Feel free to ignore while this is a draft.

This early feedback is actually useful! Thanks

Reply:
Yea I think this is a good idea. There's at least one place where the logical integer is needed (in `TextSize` [here](https://github.com/wikimedia/wikipedia-kaios/pull/300/files?file-filters%5B%5D=.js#diff-f841ba9b331ef3886506345a5e90467e92c54d03db86545b229d713a6d70ad67R31-R32)), but I've just adjusted to parse for the integer within that component